dts/bindings/cpu/qemu,virt.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2024 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: QEMU virt machine CPU node
+
+compatible: "qemu,virt"
```

Comment:
on a second thought, shouldn't this be named `qemu-virt-riscv` (or sth like that). `qemu,virt` is somehow ambiguous

Reply:
Done, I've changed the compat name to `qemu,riscv-virt`